docs/conf.py.in

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8 -*-
+#
+# @PROJECT_NAME@ documentation build configuration file, created by
+# sphinx-quickstart on Wed Jun  8 08:00:30 2016.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+import subprocess
+# sys.path.insert(0, os.path.abspath('.'))
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
```

Comment:
I think it would be good to clean out the comments that do not add any value, sush as the one above, to make the conf.py file more readable.